webpack.config.js

```diff
@@ -87,8 +88,8 @@ const config = {
                         destination: path.resolve(__dirname, 'dist/src/stepFunctions/asl/aslServer.js'),
                     },
                     {
-                        source: path.resolve(__dirname, 'dist/aslServer.js.LICENSE'),
-                        destination: path.resolve(__dirname, 'dist/src/stepFunctions/asl/aslServer.js.LICENSE'),
+                        source: path.resolve(__dirname, 'dist/aslServer.js.LICENSE.txt'),
+                        destination: path.resolve(__dirname, 'dist/src/stepFunctions/asl/aslServer.js.LICENSE.txt'),
```

Comment:
This change is not related in anyway to the features that are added here.

But this showcases the main reason why I chose to use the "`copy-webpack-plugin`" instead of "`filemanager-webpack-plugin`" for copying our artifacts for the language server into the build.

The problem with the "`filemanager-webpack-plugin`" is that it is not failing the build if a resource is not found. It is only silently writing a warning message which have been ignored until now. Also this plugin is quite old and no longer maintained.

Reply:
Should we remove `filemanager-webpack-plugin` ? cc @alanbo